### PR TITLE
Jira: OSDOCS-1878 Add Marketplace URL to ROSA Docs

### PR DESCRIPTION
**Pull request by kmccarron-rh**
For Rosa
Jira: https://issues.redhat.com/browse/OSDOCS-1878

Direct link to doc preview: https://deploy-preview-30685--osdocs.netlify.app/openshift-rosa/latest/rosa_getting_started/rosa-setting-up-environment.html#rosa-setting-up-environment_rosa-setting-up-environment

### CODE REVIEW

**Comment by kmccarron-rh** [comment]
Added Marketplace URL. Please review! 
@jeremyeder, @vkareh 
@okashi18 - Made changes to prerequisites and order of steps as requested. 

@yufchang - Just a heads-up, once finalized, this will require QE ack. 
Thanks all!

File: modules/rosa-setting-up-environment.adoc [hunk 1]
```diff
@@ -29,6 +29,10 @@ As part of the cluster creation process, `rosa` creates an `osdCcsAdmin` IAM use
 This user has `Programmatic` access enabled and the `AdministratorAccess` policy attached to it.
 ====
 +
+. Enable the ROSA service in the AWS Marketplace.
+.. Sign in to your AWS account.
+.. To enable ROSA, go to the link:https://console.aws.amazon.com/rosa/[ROSA service] and select Enable.
```
Comment (sfortner-RH) [inline modules/rosa-setting-up-environment.adoc:34]: One minor edit Enable -> *Enable*.

**Review by sfortner-RH — CHANGES_REQUESTED** [review]
One minor edit. Please make the change and squash or amend your commit.

**Review by sfortner-RH — APPROVED** [review]
LGTM! Thanks!

**Comment by kmccarron-rh** [comment]
@sfortner-RH - QE ack received in the card. Looks like we are good to merge!

**Pull request merged**